extensions/ManiphestExcelDefaultIncludeCustomFieldsFormat.php

```diff
@@ -25,35 +24,35 @@
  *   does seem to have some methods for setting/requiring a "viewer".
  *   This was not needed in the moment so it's ignored for the time being.
  */
-
-final class ManiphestExcelDefaultIncludeCustomFieldsFormat extends ManiphestExcelFormat {
+final class ManiphestExcelDefaultIncludeCustomFieldsFormatExtended extends ManiphestExcelFormat {
   public function getName() {
-    return pht('Default with Custom Fields');
+    return pht('Default with Custom Fields - extended');
   }
-
   public function getFileName() {
     return 'maniphest_tasks_'.date('Ymd');
   }
-
   public function buildWorkbook(
     PHPExcel $workbook,
     array $tasks,
     array $handles,
     PhabricatorUser $user) {
-
     $sheet = $workbook->setActiveSheetIndex(0);
     $sheet->setTitle(pht('Tasks'));
-
     // Header Cell
     // title => the displayed header title in the spreadsheet, in row 0
     // width => initial width in pixels for the column, null leaves unspecified
     // celltype => which format the column data should be set as, default is STRING
     //   can be null if it's a date field
     // isDate => there is no date format in the PHPExcel_Cell_DataType, so this is needed
     // cftype => the custom field data type, only specified for custom field headers
-
     $colHeaders = array(
       array(
+        'title' => pht('URI'),
```

Comment:
I put first and later even made column thinner (almost disappearing). Personally I would remove (Txxx is enough as reference if needed). Depends on you needs and taste ! :)